### ISSUE TITLE
crash report when enter the compact machine

### ISSUE THREAD

**Issue by runescapejon** [thraaawn/CompactMachines#193]
like the Title said when ever i enter the compact machine i crash, this is happening to random people anyways here is the crash report http://pastebin.com/QZsvzw2X

**Comment by Dooglesmile** [comment]
Hi, I'm getting the same error whenever I enter any compact machine on my server too.
I then created a single player world and tested it. The first time, it let me enter the compact machine and exit again. The second time on the same world it crashed out.

My crashlog is here: [http://pastebin.com/gXCy5Ymx]

I have recently updated a few mods in my modpack and didn't get the crash beforehand.
Comparing my own mods and those of runescapejon against the mods I've recently updated in my pack, it looks like it is possibly caused by one or more of the following mods:
CoFH Core 1.7.10R3.1.3
ThermalFoundation 1.7.10R1.2.5
ThermalExpansion 1.7.10R4.1.4
CarpentersBlocks 3.3.8.1
DraconicEvolution 1.0.2h
dsurround 1.0.5.9
EnderIO 1.7.10-2.3.0.429_beta
GalacticraftCore 3.0.12
GalacticraftMars 3.0.12
Micdoodlecore 3.0.12
Mekanism 9.1.0
MekanismGenerators 9.1.0
MekanismTools 9.1.0
ThermalDynamics 1.7.10R1.2.0
aobd 2.9.2

I shall do some testing my end to see which of these is the culprit in my instance.

**Comment by jon100456** [comment]
dang, getting the same problem @Dooglesmile  i have some of those mods please tell me which one is the culprit

**Comment by DeadSix27** [comment]
@jon100456 @runescapejon He unfortunately doesn't support 1.7.10 anymore afaik. See: https://github.com/thraaawn/CompactMachines/issues/192#issuecomment-282719988

**Comment by Dooglesmile** [comment]
Ooh I missed Optifine, we both have version 1.7.10_HD_U_D6 which I also updated the other day.
So far I've tested:
CoFH Core, Thermal dynamics, expansion and foundation.
Mekanims, Mekanism Generators and Tools
Galacticraft, planets and Micdoodlecore
Draconic Evolution.
All still crash out with the originally working versions running.
It is a pity it takes so long to load the game when it is modded!
